### PR TITLE
Add cost-saving help for the kubernetes deployment

### PR DESCRIPTION
**Pull request by pecigonzalo**
It was highlighted to us that some customers get a bit surprised by the high estimates provided for our Kubernetes deployment type and unlike the Docker Compose option, it did not provide a helper text to indicate how to reduce those resources.

I added some draft text on how we could suggest they action Kubernetes to reduce the initial costs.

Thread: https://sourcegraph.slack.com/archives/C015P61EXV1/p1610403037127800

> I just was on a call with someone who got freaked out by the estimated total CPUs for a k8s deployment, and because he'd gone there immediately (as opposed to "estimated" or "docker-compose" on the resource calculator) he didn't see our suggestion for reducing costs.

Recording: https://share.getcloudapp.com/2Nu0DvrK

/cc @emchap @slimsag

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,13 +1,11 @@
 module github.com/sourcegraph/resource-estimator
 
-go 1.13
```
Comment (daxmc99) [inline go.mod:3]: I think it's safe to bump this to 1.15, I was able to run the with it

File: go.mod [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
+go 1.14
 
 require (
-	github.com/gopherjs/vecty v0.0.0-20200301031729-eafd720f09ee
+	github.com/hexops/vecty v0.6.0
 	github.com/microcosm-cc/bluemonday v1.0.2
 	github.com/russross/blackfriday v1.5.2
```
Comment (daxmc99) [inline go.mod]: I see `github.com/pmezard/go-difflib v1.0.0 // indirect` added when I run `go mod tidy`

File: go.mod (hunk 1 above)
Comment (daxmc99) [reply go.mod:3]: Might also be nice to run `go fmt ./...` across the entire project

**Review by daxmc99 — APPROVED** [review]

**Comment by emchap** [comment]
@pecigonzalo Thank you for doing this! :)

File: go.mod (hunk 1 above)
Comment (pecigonzalo) [reply go.mod:3]: Done

File: go.mod (hunk 2 above)
Comment (pecigonzalo) [reply go.mod]: Done, I was under the impression VSCode was doing for me 🤷

**Comment by daxmc99** [comment]
Looks good, I think the last thing to do is upload this to the bucket

**Comment by emidoots** [comment]
Looks great to me, thank you for doing this!

**Pull request merged**